hedera-node/hedera-mono-service/src/test/java/com/hedera/services/store/contracts/precompile/TransferPrecompilesTest.java

```diff
@@ -1402,12 +1409,123 @@ void gasRequirementReturnsCorrectValueForTransferSingleToken() {
 
         subject.prepareFields(frame);
         subject.prepareComputation(input, a -> a);
+
         long result = subject.getPrecompile().getGasRequirement(TEST_CONSENSUS_TIME);
 
         // then
         assertEquals(EXPECTED_GAS_PRICE, result);
     }
 
+    @Test
+    void minimumFeeInTinybarsHbarOnlyCryptoTransfer() {
```

Comment:
Could you add test case for multiple hbar transfers (List<SyntheticTxnFactory.HbarTransfer> hbarTransfers > 1)?

Reply:
done